src/inc/winwrap.h

```diff
@@ -723,12 +684,62 @@
 #define _T(str) W(str)
 #endif
 
-
 // on win98 and higher
 #define Wszlstrlen      lstrlenW
 #define Wszlstrcpy      lstrcpyW
 #define Wszlstrcat      lstrcatW
 
+//File and Directory Functions which need special handling for LongFile Names
+//Note only the functions which are currently used are defined
+#define WszLoadLibrary         LoadLibraryExWrapper
+#define WszLoadLibraryEx       LoadLibraryExWrapper
+#define WszCreateFile          CreateFileWrapper
+#define WszSetFileAttributes   SetFileAttributesWrapper  
+#define WszGetFileAttributes   GetFileAttributesWrapper
+#define WszGetFileAttributesEx GetFileAttributesExWrapper
+#define WszDeleteFile          DeleteFileWrapper
+#define WszFindFirstFileEx     FindFirstFileExWrapper
+#define WszFindNextFile        FindNextFileW
+#define WszCopyFile            CopyFileWrapper
+#define WszCopyFileEx          CopyFileExWrapper
+#define WszMoveFile            MoveFileWrapper
+#define WszMoveFileEx          MoveFileExWrapper
+
+#define WszSearchPath          SearchPathWrapper
+#define WszGetShortPathName    GetShortPathNameWrapper
+#define WszGetLongPathName     GetLongPathNameWrapper
+
+#define WszCreateDirectory     CreateDirectoryWrapper 
+#define WszRemoveDirectory     RemoveDirectoryWrapper
+
+#define WszCreateHardLink      CreateHardLinkWrapper
+#define WszCreateSymbolicLink  CreateSymbolicLinkWrapper
+#define WszGetBinaryType       GetBinaryTypeWrapper
+
+//Can not use extended syntax 
+#define WszGetFullPathName     GetFullPathNameW
+#define WszGetCurrentDirectory GetCurrentDirectoryW
+#define WszGetTempFileName     GetTempFileNameW
+#define WszGetTempPath         GetTempPathW
+
+#if FEATURE_PAL
+#define WszFindFirstFile     FindFirstFileW
```

Comment:
> The Apis which have an out parameter for path will have the wrappers

Are you saying that Windows version of the same API takes out argument and thus, needs a wrapper but PAL implementation does not?